docs/source/upgrade.rst

```diff
@@ -31,7 +31,7 @@ Dropping Support of Python 3.4
 ------------------------------
 
 In Chainer v6, Python 3.4 is no longer supported as it reaches its end-of-life (EOL) in March 2019.
-Python 3.5.1 is the minimum Python 3 version supported by Chainer v6.
+Python 3.5.2 is the minimum Python 3 version supported by Chainer v6.
```

Comment:
Let's avoid changing requirements for v6 (1) It is a stable version which is already released (2) onnx-chainer is not backported to v6

Reply:
Thank you for pointing out! Should be fixed with: https://github.com/chainer/chainer/pull/8410/commits/38a9f186b2c4f79f7987cb95cdb5c645c2615606